### ISSUE TITLE
error:para_RepeatMasker

### ISSUE THREAD

**Issue by changchuanjun** [chenlianfu/geta#33]
failed to execute: para_RepeatMasker --out_prefix RepeatModeler_out --lib RM_*/*.classified --cpu 4 --tmp_dir para_RepeatMasker.tmp /home/changchuanjun/gene_structure_annotation_20240224/Fuji_Ral-Fuji_Del_20240226/Fuji_Ral_Hap1/out.tmp/genome.fasta &> para_RepeatMasker.log
![1708945984474](https://github.com/chenlianfu/geta/assets/155738984/2806ef33-8632-46ac-b5ca-ffaa65581dde)
I want to ask how to solve this problem?

**Comment by qdu-beep** [comment]
Hello，
Has your problem been resolved? I encountered the same problem. Hope you can help me if you solved this problem.

**Comment by changchuanjun** [comment]
@qdu-beep ,No, I have not solved this problem. I changed  to other pipeline to annotation the gene structure. I used BRAKER3 to annotate my genomes with the RNASeq data and homologous protein data .This is the reference about BRAKER3:

BRAKER3: Fully automated genome annotation using RNA-Seq and protein evidence with GeneMark-ETP, AUGUSTUS and TSEBRA
Lars Gabriel, Tomáš Brůna, Katharina J. Hoff, Matthis Ebel, Alexandre Lomsadze, Mark Borodovsky, Mario Stanke
bioRxiv 2023.06.10.544449; doi: https://doi.org/10.1101/2023.06.10.544449

**Comment by caiisen** [comment]
same problem

**Comment by caiisen** [comment]
仔细研究了一下，这个报错的原因是缺少consensi.fa.classified这个文件，它本应该在1.RepeatMasker/repeatModeler/RM开头的文件夹中，所以应该去检查RepeatModeler.log。在我的案例中，最后RepeatClassifier的步骤提示Missing Libraries/RepeatPeps.lib.psq。这样一来就明白了，在Libraries里找不到RepeatPeps，导致RepeatModeler无法进行RepeatClassifier，没有生成consensi.fa.classified。

我是通过conda安装的RepeatModeler，它的Libraries默认在conda/share/RepeatModeler里，但是日志里显示在conda/bin里找不到Libraries。所以尝试在bin里添加一个软连接到share/RepeatModeler/Libraries，问题就解决了。

**Comment by changchuanjun** [comment]
Thanks for your information👍🏻👍🏻👍🏻



---Original---
From: "Cai ***@***.***&gt;
Date: Wed, Oct 30, 2024 14:24 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [chenlianfu/geta] error:para_RepeatMasker (Issue #33)




 
仔细研究了一下，这个报错的原因是缺少consensi.fa.classified这个文件，它本应该在1.RepeatMasker/repeatModeler/RM开头的文件夹中，所以应该去检查RepeatModeler.log。在我的案例中，最后RepeatClassifier的步骤提示Missing Libraries/RepeatPeps.lib.psq。这样一来就明白了，在Libraries里找不到RepeatPeps，导致RepeatModeler无法进行RepeatClassifier，没有生成consensi.fa.classified。
 
我是通过conda安装的RepeatModeler，它的Libraries默认在conda/share/RepeatModeler里，但是日志里显示在conda/bin里找不到Libraries。所以尝试在bin里添加一个软连接到share/RepeatModeler/Libraries，问题就解决了。
